projectionizer/fiber_simulation.py

```diff
@@ -200,7 +203,6 @@ def get_l5_l34_border_voxel_indices(atlas, regions):
 def get_fiber_directions(fiber_positions, atlas):
     """Get the fiber directions at positions defined in fiber_positions."""
     orientation = atlas.load_data("orientation", cls=voxcell.OrientationField)
-    orientation.raw = orientation.raw.astype(np.int8)
```

Comment:
Hmm, interesting; at one point `char`s where supported for orientations (https://bbpteam.epfl.ch/project/issues/browse/NDM-2865);
![image](https://github.com/user-attachments/assets/38113b3e-2672-43f8-9a54-0e0e4d572cb6)

I wonder if it's related to that?  Perhaps some of the orientations don't fit this?

Reply:
According to https://voxcell.readthedocs.io/en/latest/orientation.html type can be float (default) or int8 (optional).

The class `OrientationField` supports both at https://github.com/BlueBrain/voxcell/blob/d42b95ed76d83dfac22f7b33c0368a8bdbe9be88/voxcell/voxel_data.py#L371

In case of int8, the values are divided by 127 in the lookup method:
https://github.com/BlueBrain/voxcell/blob/d42b95ed76d83dfac22f7b33c0368a8bdbe9be88/voxcell/voxel_data.py#L388-L390

So I think it's safe to let voxcell handle the type.